website/content/docs/security/acl/acl-roles.mdx

```diff
@@ -53,28 +54,126 @@ $ curl --request PUT --data @payload.json http://127.0.0.1:8500/v1/acl/role
 
 Refer to the [API documentation](/consul/api-docs/acl/roles) for details.
 
-## Role Attributes
+## Role attributes
 
 Roles may contain the following attributes:
 
 - `ID`: The `ID` is an auto-generated public identifier. You can specify the role `ID` when linking it to tokens.
 - `Name`: A unique meaningful name for the role. You can specify the role `Name` when linking it to tokens.
 - `Description`: (Optional) A human-readable description of the role.
 - `Policies`: Specifies a the list of policies that are applicable for the role. The object can reference the policy `ID` or `Name` attribute.
+- `TemplatedPolicies`: Specifies a list of templated polcicies that are applicable for the role. See [Templated Policies](#templated-policies) for details.
 - `ServiceIdentities`: Specifies a list of services that are applicable for the role. See [Service Identities](#service-identities) for details.
 - `NodeIdentities`: Specifies a list of nodes that are applicable for the role. See [Node Identities](#node-identities) for details.
 - `Namespace`: <EnterpriseAlert inline /> The namespace that the policy resides in. Roles can only be linked to policies that are defined in the same namespace. See [Namespaces](/consul/docs/enterprise/namespaces) for additional information. Requires Consul Enterprise 1.7.0+
 - `Partition`: <EnterpriseAlert inline/> The admin partition that the policy resides in. Roles can only be linked to policies that are defined in the same admin partition. See [Admin Partitions](/consul/docs/enterprise/admin-partitions) for additional information. Requires Consul Enterprise 1.10.0+.
 
-## Service Identities
+## Templated policies
+
+You can specify a templated policy when configuring roles or linking tokens to policies. Templated policies enable you to quickly construct policies for common consul use cases, rather than creating identical policies for each use cases.
+
+Templated policies are used during the authorization process to automatically generate a policy for the use case specified. Consul links the generated policy to the role or token so that it will have permission for the specific use case.  
```

Comment:
```suggestion
Consul uses templated policies during the authorization process to automatically generate a policy for the use case specified. Consul links the generated policy to the role or token so that it will have permission for the specific use case.  
```